hopp/simulation/technologies/financial/custom_financial_model.py

```diff
@@ -395,7 +395,7 @@ def annual_energy(self) -> float:
     
     @property
     def om_total_expense(self) -> float:
-        return self.value('om_total_expense')
+        return self.Outputs.om_total_expense
```

Comment:
Where are values being assigned to the Outputs class?

Reply:
That's a good question. Upon further investigation it doesn't get assigned. I mirrored the structure in the `lcoe_real` property since the naming convention switches between `HybridSimulation` and `PowerSource` and `CustomFinancialModel` so that it returns `om_total_expense` calculated in `PowerSource`

Reply:
It's true that there's a lot of functionality that hasn't been implemented in this class yet.

Reply:
So for the Outputs class, not all of the attributes in that class are being assigned. Think of it as the framework (based on PySAM nomenclature) that can be filled in as we need.

Reply:
Exactly!